docs/app-modules/interchain-accounts/transactions.md

```diff
@@ -0,0 +1,21 @@
+<!--
+order: 5
+-->
+
+# Transactions
+
+Learn about Interchain Accounts transaction execution {synopsis}
+
+## Executing a transaction
+
+As described in [Authentication Modules](./auth-modules.md#trysendtx) transactions are executed using the interchain accounts controller API and require a `Base Application` as outlined in [ICS30 IBC Middleware](https://github.com/cosmos/ibc/tree/master/spec/app/ics-030-middleware) to facilitate authentication. The method of authentication remains unspecified to provide flexibility for the authentication module developer.
```

Comment:
Is this correct `[Authentication Modules](./auth-modules.md#trysendtx)`?

Reply:
Worked when I was running a local server with docs. Link checker is happy too